skimage/filters/ridges.py

```diff
@@ -197,26 +197,26 @@ def meijering(image, sigmas=range(1, 10, 2), alpha=None,
     for i, sigma in enumerate(sigmas):
 
         # Calculate (sorted) eigenvalues
-        eigenvalues = compute_hessian_eigenvalues(image, sigma, sorting='val')
+        eigenvalues = compute_hessian_eigenvalues(image, sigma, sorting='abs')
 
         if ndim > 1:
 
             # Set coefficients for scaling eigenvalues
             coefficients = [alpha] * ndim
             coefficients[0] = 1
 
-            # Compute auxiliary variables l_i = e_i + sum_{j!=i} alpha * e_j
+            # Compute normalized eigenvalues l_i = e_i + sum_{j!=i} alpha * e_j
             auxiliary = [np.sum([eigenvalues[i] * np.roll(coefficients, j)[i]
                          for j in range(ndim)], axis=0) for i in range(ndim)]
 
-            # Compute maximum over auxiliary variables
-            auxiliary = np.max(auxiliary, axis=0)
+            # Get maximum eigenvalues by magnitude
+            auxiliary = auxiliary[1]
```

Comment:
should this be 1 or 0?

Reply:
Thanks for looking through this carefully! 1 is correct here since auxiliary is an array of eigenvalues sorted by magnitude and we want the larger ones. I attached a screenshot from the original paper, also to help answering the second question.
![2019-06-05-092847_1680x1050_scrot](https://user-images.githubusercontent.com/29574653/58938696-b474fd00-8775-11e9-9458-d95593d66878.png)

Reply:
Maybe use `-1`? Even if this is a 2D-only algorithm, I grate at 2D-specific magic values. =)

Reply:
Oh, you are so right, thanks! :)